src/routes/projects/new.svelte

```diff
@@ -229,6 +227,17 @@
     } catch (e) {
       console.error(e);
     }
+
+    prepareAfterBackend(updatedActivity);
+    for (let activity of $project.activities) {
+      if (activity._type === ActivityTypes.TEMPLATE && activity.timeframe == null) {
+        activity.timeframe = {
+          start: updatedActivity.timeframe.start,
+          end: updatedActivity.timeframe.end,
```

Comment:
Could be simplified to 
```suggestion
          ...updatedActivity.timeframe,
```
but not a big deal

Reply:
Meh, let's just be explicit